connectors/sources/sharepoint_server.py

```diff
@@ -689,15 +686,14 @@ async def get_docs(self, filtering=None):
         Yields:
             dictionary: dictionary containing meta-data of the SharePoint objects.
         """
-
         server_relative_url = []
 
-        for collection in self.sharepoint_client.site_collections_path:
-            server_relative_url.append(collection)
+        for collection in self.sharepoint_client.site_collections:
+            server_relative_url.append(f"{collection}")
             async for site_data in self.sharepoint_client.get_sites(
-                site_url=collection
+                site_url=f"{collection}"
             ):
-                server_relative_url.append(site_data["ServerRelativeUrl"])
+                server_relative_url.append(site_data["Url"])
```

Comment:
I'd rename `server_relative_url` to `server_url` cause it's not relative any more